examples/cloudproviders/k8s/rbac.yaml

```diff
@@ -0,0 +1,26 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: gostatsd
+rules:
+  - apiGroups: [""]
+    resources:
+      - pods
+    verbs:
+      - get
```

Comment:
If we only use watch then we don't need get.

Reply:
True, I'll remove it.